Document/GroupManager.php

```
@@ -30,4 +30,17 @@ public function findGroupsBy(array $criteria = null, array $orderBy = null, $lim
     {
         return $this->repository->findBy($criteria, $orderBy, $limit, $offset);
     }
-}
\ No newline at end of file
+
+    /**
+     * @param array $criteria
+     * @param integer $page
+     * @param integer $limit
+     * @param array $sort
+     *
+     * @return \Sonata\DatagridBundle\Pager\PagerInterface
+     */
+    public function getPager(array $criteria, $page, $limit = 10, array $sort = array())
+    {
+        // TODO: Implement getPager() method.
```

Can you throw an Exception: new \RuntimeException('method not implemented');